api/mailer/SceGoogleApiHandler.js

```diff
@@ -147,6 +147,34 @@ class SceGoogleApiHandler {
       }
     });
   }
-}
 
-module.exports = { SceGoogleApiHandler };
+  /**
+   * Grabs all the events for a given calendar by its id.
+   * will log the next 10 events into the console.
+   * @param calendarId {string} calendar id for which calendar to pull from
+   */
```

Comment:
specify what the funciton returns here
```suggestion
   * @returns {{Array<Object>|Error)} The calendar events from Google or an error.
   */
```